benchmarks/benchmarks/ci/esmf_regridder.py

```diff
@@ -37,3 +41,45 @@ def time_make_grid(self):
         esmf_grid.data[:] = 0
 
     time_make_grid.version = 1
+
+
+class TimeRegridding:
+    params = ["similar", "large source", "large target", "mixed"]
```

Comment:
I'm not sure what name you might use, but are you aware that you could name this `param` by using the `param_names` variable? This makes things more readable when viewing published results.